config/Substfiles.am

```diff
@@ -6,6 +6,7 @@ subst_sed_cmd = \
 	-e 's|@runstatedir[@]|$(runstatedir)|g' \
 	-e 's|@initconfdir[@]|$(initconfdir)|g' \
 	-e 's|@initdir[@]|$(initdir)|g' \
+	-e 's|@initramfsdir[@]|$(initramfsdir)|g' \
```

Comment:
I don't know if this is needed and what actually does, so please correct me here.